src/components/shared/TwoButtonModal.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './TwoButtonModal.scss';
+
+const Modal = ({ isShowing, hide, title, text, handleConfirm }) =>
+  isShowing
+    ? 
+        <>
```

Comment:
why is this needed if you are already using a single tag surrounding everything else?

Reply:
no need will remove tnx